test/test_images/sendeventswithtracing/main.go

```diff
@@ -0,0 +1,174 @@
+/*
```

Comment:
Can we reuse the `sendevents` image, by adding an arg like `add-tracing`? Seems most code here is duplicate.

Reply:
Yes good idea